### DISCUSSION TITLE
Is it possible to remove the add action from a repeater?

### DISCUSSION THREAD

**Discussion by MtDalPizzol** [filamentphp/filament#10128] [Help] [answered]
Hi folks. Couldn't find this info in the docs. Found how to  [Customize the repeater item actions](https://filamentphp.com/docs/3.x/forms/fields/repeater#customizing-the-repeater-item-actions), but that didn't help.

I'm using a repeater in a form where the user is only allowed to edit the related records in the repeater, but can't add new ones.

**Comment by leandrocfe** [comment, marked as answer]
this?
https://filamentphp.com/docs/3.x/forms/fields/repeater#preventing-the-user-from-adding-items

**Reply by MtDalPizzol** [reply]
Damn. I was so focused on the Eloquent relationships section that I missed that. Thank you!

**Reply by leandrocfe** [reply]
Have you worked with Inertia? I think I've ever seen you talking about vuejs and inertia in pt-br...

**Reply by MtDalPizzol** [reply]
Yup! That's probably me! :sweat_smile:

Giving LW a second chance since the major objections I had with it seem to be solved in V3.

**Reply by leandrocfe** [reply]
Nice, welcome!
We have a filament community in Brazil https://www.youtube.com/@filamentbr

Discord
https://discord.com/channels/883083792112300104/966832715536162846

**Reply by MtDalPizzol** [reply]
Awesome! Subscribed! 😉
